Cassette/CASFileObjectQueue.h

```diff
@@ -21,12 +21,14 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  * Initializes an @c CASFileObjectQueue with a file in the application's Library directory,
  * returning nil if there was an error.
+ * Note: If target and intermediary directories do not exist in path they will be created.
```

Comment:
I think we should uplevel this handling to the application. This feature increases the scope of what this library provides. What do you think?